File: WeatherApp.swift
import Foundation

func getWeather(city: String) -> String {
    let apiKey = "YOUR_API_KEY"
    let urlString = "http://api.openweathermap.org/data/2.5/weather?q=\(city)&appid=\(apiKey)"
    
    guard let url = URL(string: urlString) else {
        return "Error: Invalid URL"
    }
    
    let data = try? Data(contentsOf: url)
    if let jsonData = data {
        if let json = try? JSONSerialization.jsonObject(with: jsonData, options: []) as? [String: Any],
           let main = json["main"] as? [String: Any],
           let weather = (json["weather"] as? [[String: Any]])?.first {
            
            let temperature = main["temp"] as? Double ?? 0.0
            let description = weather["description"] as? String ?? "No description available"
            return "Temperature: \(temperature)°C\nWeather: \(description)"
        }
    }
    return "Error: Unable to fetch data"
}

print(getWeather(city: "London"))

File: WeatherApp.kt
import java.net.HttpURLConnection
import java.net.URL
import org.json.JSONObject

fun getWeather(city: String): String {
    val apiKey = "YOUR_API_KEY"
    val urlString = "http://api.openweathermap.org/data/2.5/weather?q=$city&appid=$apiKey"
    val url = URL(urlString)
    val connection = url.openConnection() as HttpURLConnection
    connection.requestMethod = "GET"
    val response = connection.inputStream.bufferedReader().readText()
    val data = JSONObject(response)

    return if (data.getInt("cod") == 200) {
        val temperature = data.getJSONObject("main").getDouble("temp")
        val description = data.getJSONArray("weather").getJSONObject(0).getString("description")
        "Temperature: $temperature°C\nWeather: $description"
    } else {
        "Error: Unable to fetch data"
    }
}

fun main() {
    println(getWeather("London"))
}